src/interface/src/app/plan/plan-summary/saved-scenarios/saved-scenarios.component.spec.ts

```diff
@@ -91,6 +91,7 @@ describe('SavedScenariosComponent', () => {
     component = fixture.componentInstance;
 
     component.plan = MOCK_PLAN;
```

Comment:
nit- instead of doing ` component.plan.permissions =  ['add_scenario']` maybe you can extend MOCK_PLAN
`component.plan = {...MOCK_PLAN, permissions: ['add_scenario']}`

Reply:
Yes! I can't believe I missed this opportunity to use spread...